homu/server.py

```diff
@@ -1,7 +1,8 @@
 import hmac
 import json
 import urllib.parse
-from .main import PullReqState, parse_commands, db_query, INTERRUPTED_BY_HOMU_RE, synchronize
+from .main import (PullReqState, parse_commands, db_query,
+                   INTERRUPTED_BY_HOMU_RE, synchronize)
```

Comment:
Adding only a line break means you get unnecessary diff noise if you change the function name. I've gotten used to block indent so the `):` without an indent looks fine to me, but I also think that there are further improvements to be had in the future by introducing some intermediate data structures as appropriate and reducing the general spaghetti-ness in further refactors.